openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -955,17 +955,20 @@
 - control_key: AC-6 (9)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        The customer will be responsible for auditing the execution of
-        privileged functions. A successful control response will need
-        to relate this requirement to the general auditing requirements
-        of the AU control family.
+        OpenShift performs audit logging by default, which makes this
+        control inherently met.
 
-        A control response is planned. Progress can be tracked via:
+        What constitutes an action by an admin role must be synthetized in a
```

Comment:
@JAORMX @carlosmmatos would appreciate a second pair of eyes here. AC-6(9) asks for auditing of privileged functions. OCP audits all requests by default, but which of them are privileged must be synthetized from the request by SIEM or otherwise.

The logs would contain something like this:
```
authorization.k8s.io/decision":"allow","authorization.k8s.io/reason":"RBAC: allowed by ClusterRoleBinding \"system:openshift:operator:authentication\" of ClusterRole \"cluster-admin\" to ServiceAccount \"authentication-operator/openshift-authentication-operator>
```

Reply:
I would say that the context of privileged functions is whatever type of administrative action (probably defined by the clusteradmin role) can be taken upon the system.. ie changing settings, user settings, etc

As long as OCP audits 'all' then we should be good. I don't think this control really asks about SIEM usage in the sense of viewing the audit logs.. just that OCP does audit these events.